x-pack/plugins/event_log/server/es/cluster_client_adapter.ts

```diff
@@ -129,84 +130,106 @@ export class ClusterClientAdapter {
 
   public async queryEventsBySavedObject(
     index: string,
+    namespace: string | undefined, // TODO - add term clause for namespace
     type: string,
     id: string,
     { page, per_page: perPage, start, end, sort_field, sort_order }: FindOptionsType
   ): Promise<QueryEventsBySavedObjectResult> {
-    try {
-      const {
-        hits: { hits, total },
-      }: SearchResponse<unknown> = await this.callEs('search', {
-        index,
-        // The SearchResponse type only supports total as an int,
-        // so we're forced to explicitly request that it return as an int
-        rest_total_hits_as_int: true,
-        body: {
-          size: perPage,
-          from: (page - 1) * perPage,
-          sort: { [sort_field]: { order: sort_order } },
-          query: {
-            bool: {
-              must: reject(
-                [
-                  {
-                    nested: {
-                      path: 'kibana.saved_objects',
-                      query: {
-                        bool: {
-                          must: [
-                            {
-                              term: {
-                                'kibana.saved_objects.rel': {
-                                  value: SAVED_OBJECT_REL_PRIMARY,
-                                },
-                              },
+    const defaultNamespaceQuery = {
```

Comment:
These are the new query DSL clauses to ensure we only return SO's in the relevent space - likely not an issue now, but will be once we can query pre-configured actions - if you could query those now (you can't), the query would return events across all spaces, without these additional clauses.

The rule with namespaces is that they are `string | undefined` (`undefined` for the default space), and that's how we "store" them in ES - the SO reference in the event will not have a `namespace` field for the default space.